plugins/markdown/src/org/intellij/plugins/markdown/ui/preview/StyleOverrides.kt

```diff
@@ -0,0 +1,90 @@
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package org.intellij.plugins.markdown.ui.preview
+
+import com.intellij.openapi.editor.colors.EditorColors
+import com.intellij.openapi.editor.colors.EditorColorsManager
+import com.intellij.util.ui.UIUtil
+import java.awt.Color
+
+private fun Color.webRgba() = "rgba($red, $green, $blue, $alpha)"
+
+/**
+ * Simple linear contrast function.
+ *
+ * 0 < coefficient < 1 results in reduced contrast.
+ * coefficient > 1 results in increased contrast.
+ */
+private fun Color.contrast(coefficient: Double) =
+  Color(
+    (coefficient * (red - 128) + 128).toInt(),
+    (coefficient * (green - 128) + 128).toInt(),
+    (coefficient * (blue - 128) + 128).toInt(),
+    alpha
+  )
+
+/**
+ * This method will generate stylesheet with colors, matching current IDE colors,
+ * for markdown elements. Generated rules will override base rules from the
+ * default.css and darcula.css, so the preview elements will have correct (almost) colors.
```

Comment:
If you write `(almost)` -- you should explain in which cases it would not generate correct colors :)